src/stories/Block.stories.tsx

```diff
@@ -0,0 +1,168 @@
+// Block Story
+
+import { Story, Meta } from "@storybook/react/types-6-0";
+import { Action } from "../components/Action";
+import { Actions } from "../components/Actions";
+import { Block, BlockProps } from "../components/Block";
+import { blogPosts, experience, openSource, projects, spotify } from "./_data";
+
+export default {
+  title: "UI/Block",
+  component: Block,
+} as Meta;
+
+const exampleTitleHanlder = (title: string) => {
+  console.log(title);
+}
+
+const exampleChangeSize = (size: number) => {
+  console.log("size: " + size);
+}
+
+const Template: Story<BlockProps> = (args) => {
+  return <div className="flex flex-col gap-8 max-w-screen-xl px-24 mx-auto">
+    <div className="w-full">
+      <Block {...args} size={3} isEditable={true}/>
+    </div>
+    <div className="flex gap-8 flex-col md:flex-row">
+      <div className="w-full md:w-[calc((100%_-_32px)_/_2_*_1)] xl:w-[calc((100%_-_64px)_/_3_*_2_+_32px)]">
+        <Block {...args} size={2} isEditable={true}/>
+      </div>
+      <div className="w-full md:w-[calc((100%_-_32px)_/_2_*_1)] xl:w-[calc((100%_-_64px)_/_3_*_1)]">
+        <Block {...args} size={1} isEditable={true}/>
+      </div>
+    </div>
+  </div>
```

Comment:
@NilsJacobsen This wrapper should be it's own component, what do you think?

Reply:
Jap you're right let my isolate this.